config/routes.rb

```diff
@@ -13,6 +13,7 @@
   resources :identities, only: %i[index show] do
     member do
       get :show_sites
+      post :toggle_portal
```

Comment:
great work adding a post route for a toggle. As we add other features and want to toggle those features as well, are we going to be able to reuse this route or will we have to add a new route for every new feature?

Reply:
I believe we can reuse it as long as we logged in (have accessed `identity`, which is our token when are are logged in for `FormUser` class, which inherits from user)